deposit_contract.sol

```diff
@@ -87,7 +87,7 @@ contract DepositContract is IDepositContract {
         require(msg.value >= MIN_DEPOSIT_AMOUNT);
         require(msg.value % GWEI == 0);
         uint deposit_amount = msg.value / GWEI;
-        require(deposit_amount < 2**64);
+        require(deposit_amount < type(uint64).max);
```

Comment:
technically I guess this should be `require(deposit_amount <= type(uint64).max); `

Reply:
This is an outdated branch, will fix it once the release is out and will also set the pragma fixed to that version.